app/dao/users_dao.py

```diff
@@ -3,15 +3,20 @@
 from sqlalchemy import func
 from app import db
 from app.models import (User, VerifyCode)
-
+from app import secrets
 
 def _remove_values_for_keys_if_present(dict, keys):
     for key in keys:
         dict.pop(key, None)
 
 
 def create_secret_code():
-    return ''.join(map(str, random.sample(range(9), 5)))
+    L1 = []
+    for i in range(0, 5):
+        L1.append(secrets.randbelow(10))    #return cryptographically strong random number using secrets module
+    L2 = ''.join(map(str, L1))
+    return L2
+
```

Comment:
I think this could be cleaner – what about:

`str(secrets.randbelow(10**5)).zfill(5)`